app/CreateDeal.tsx

```diff
@@ -7,19 +7,22 @@ import { PageHeader } from "./ui/PageHeader";
 import { FormLabel } from "./ui/FormLabel";
 import { FormGroup } from "./ui/FormGroup";
 import { FormControlString } from "./ui/FormControlString";
-import { Form, FormCheck, InputGroup } from "react-bootstrap";
+import { InputGroup } from "react-bootstrap";
 import { FormControlNumber } from "./ui/FormControlNumber";
+import { FormCheckRadio } from "./ui/FormCheckRadio";
 
 export function CreateDeal(props: {
   onCreateDealClick: () => void;
   onCancelClick: () => void;
 }): ReactElement {
+  const units = ["kg", "L", "unit"];
   const [errorMessage, setErrorMessage] = useState<string>("");
   const [itemName, setItemName] = useState<string>("");
   const [pricePerUnit, setPricePerUnit] = useState<string>("0");
-  const [date, setDate] = useState(new Date());
+  const [selectedUnit, setSelectedUnit] = useState(units[0]);
```

Comment:
let's have no default and make the user input so we can be sure it's what they wanted